docs/Using-Fleet/REST-API.md

```diff
@@ -5119,6 +5119,12 @@ Returns the query specified by ID.
     "name": "centos_hosts",
     "description": "",
     "query": "select 1 from os_version where platform = \"centos\";",
+    "team_id": null,
+    "interval": 3600,
+    "platform": "",
```

Comment:
I suggest we call this field `platforms_to_query` since it can represent multiple platforms. What do folks think? @zhumo @rachaelshaw

Reply:
Or maybe `platforms` since it implies there can be multiple.

Reply:
For consistency let's leave as is? (policies and labels use `"platform"` too).

But if you folks feel we should change this, am ok with it, let's change it on this PR. (We should all be taking this PR as source of truth.)

Reply:
Sounds good Lucas. Why don't we just add an explanatory comment then for any future devs working on this.

Reply:
+1, I'll make sure to add this to the three places where `"platform"` is actually a comma separated list of platforms.